<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="actionLoadURL"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_OS">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <widget class="QMenu" name="menu_Rotation">
     <property name="title">
      <string>&amp;Rotation</string>
     </property>
     <addaction name="actionVertical"/>
     <addaction name="actionHorizontal"/>
    </widget>
    <widget class="QMenu" name="menu_Screen">
     <property name="title">
      <string>&amp;Screen</string>
     </property>
     <addaction name="action320x400"/>
     <addaction name="action320x480"/>
     <addaction name="action1024x768"/>
    </widget>
    <widget class="QMenu" name="menu_OS_2">
     <property name="title">
      <string>&amp;OS</string>
     </property>
     <addaction name="actionIOS"/>
     <addaction name="actionAndroid"/>
     <addaction name="actionWebOS"/>
    </widget>
    <addaction name="menu_OS_2"/>
    <addaction name="menu_Screen"/>
    <addaction name="menu_Rotation"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInspector"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_OS"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionLoadURL"/>
   <addaction name="actionStop"/>
   <addaction name="actionReload"/>
  </widget>
  <action name="actionNewWindow">
   <property name="text">
    <string>&amp;New Window...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoadURL">
   <property name="text">
    <string>&amp;Load URL...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../qrc/mobileondesktop.qrc">
     <normaloff>:/view-refresh.png</normaloff>:/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+;</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action320x400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>320x400</string>
   </property>
  </action>
  <action name="action320x480">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>320x480</string>
   </property>
  </action>
  <action name="actionIOS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>iPhone 4</string>
   </property>
  </action>
  <action name="actionAndroid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Android 2.2</string>
   </property>
  </action>
  <action name="actionWebOS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Web OS 1.4</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../qrc/mobileondesktop.qrc">
     <normaloff>:/go-previous.png</normaloff>:/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../qrc/mobileondesktop.qrc">
     <normaloff>:/go-next.png</normaloff>:/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../qrc/mobileondesktop.qrc">
     <normaloff>:/window-close.png</normaloff>:/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Stop loading</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionInspector">
   <property name="text">
    <string>&amp;Inspector...</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action1024x768">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1024x768</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/mobileondesktop.qrc"/>
 </resources>
 <connections/>
</ui>
